src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { forwardRef } from "react";
+import {
+  chakra,
+  Tooltip as ChakraTooltip,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+import Icon from "../Icons/Icon";
+import Image from "../Image/Image";
+import ComponentWrapper from "../ComponentWrapper/ComponentWrapper";
+
+export interface TooltipProps {
+  /** Any child node passed to the component. */
+  children: React.ReactNode;
+  /** Value used to populate the tooltip content. */
+  content: string | React.ReactNode;
+  /** A class name for the Tooltip parent div. */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes. */
+  id?: string;
+  /** Adds the `disabled` prop to the Tooltip when true. */
+  isDisabled?: boolean;
+  /** Wraps the children of the tooltip in `ComponentWrapper` with `tabIndex=0` when true. */
+  shouldWrapChildren?: boolean;
+}
+
+export const Tooltip = chakra(
+  forwardRef<HTMLDivElement, TooltipProps>((props, ref?) => {
+    const {
+      children,
+      className,
+      content,
+      id,
+      isDisabled,
+      shouldWrapChildren,
+      ...rest
+    } = props;
+
+    if (typeof content !== "string") {
+      React.Children.map(
+        content as React.ReactNode,
+        (contentChild: React.ReactElement) => {
+          if (contentChild.type !== Icon || contentChild.type !== Image) {
+            console.warn(
+              "NYPL Reservoir Tooltip: Pass in a string, Icon, or Image into the 'content' prop."
```

Comment:
Can you add a test for this? Try passing in a random component into the `content` prop. Here's an example of a test: https://github.com/NYPL/nypl-design-system/blob/development/src/components/Checkbox/Checkbox.test.tsx#L247

Reply:
Oh, maybe the warning should be clearer and say "a DS Icon, or DS Image". It's possible that a user may think an `Icon` or `Image` component from another library is okay to pass. What do you think?

Reply:
I agree that would be clearer to the user. Updated.